### DISCUSSION TITLE
Is there a way to just display image using path?

### DISCUSSION THREAD

**Discussion by tarunjain1st** [Schweinepriester/github-profile-achievements#870] [Q&A]
how to add images in readme using absolute/relative path after storing image to GitHub repo.
* able to use drag option which upload image to githubuserbase, but it can't be delete

**Comment by the1Riddle** [comment]
Maybe you can try these options.

```
![ALT of the img here](the link to the image here from the stored repo)
```

note that in the above format, the link of the img could be from any online site with an image link

but when the image is not on your repo and you still want to use the image.. then you can `drag and drop` or `copy-paste` the image from where it is to an `opened issue` then use the generated link on your readme

![Capture](https://github.com/the1Riddle/the1Riddle/assets/125451537/96ecbba1-a056-45e5-8e34-28ce2451468a)

**Comment by arifpro** [comment]
`![alt text](absolute/relative path)`

example:

![arctic-code-vault-contributor-default-badge](./images/arctic-code-vault-contributor-default.png)

or

![arctic-code-vault-contributor-default-badge](../images/arctic-code-vault-contributor-default.png)

Note: it will work only on markdown. But you can use same thing from githubusercontent by repo name and branch name as well anywhere.

example:

![arctic-code-vault-contributor-default-badge](https://raw.githubusercontent.com/Schweinepriester/github-profile-achievements/main/images/arctic-code-vault-contributor-default.png)


Please let me know if you need more information @tarunjain1st bro
